src/main/java/appeng/core/definitions/AEItems.java

```diff
@@ -60,6 +60,7 @@
 import appeng.items.tools.BiometricCardItem;
 import appeng.items.tools.MemoryCardItem;
 import appeng.items.tools.NetworkToolItem;
+import appeng.items.tools.fluix.*;
```

Comment:
I am actually surprised the build didn't barf at you, spotless should not have allowed wildcard imports 🤔

Reply:
I think IntelliJ did that on its own for whatever reason when I got done importing all the new tool classes instead of just leaving them as one per line. I'll sort that out real quick.